plugins/wayback.py

```diff
@@ -10,58 +10,61 @@
 # Updated at: 2024-03-13
 #
 
+# 필요한 라이브러리와 모듈을 임포트합니다.
 import requests
-
 from server import Extension
+from decouple import config
```

Comment:
Please explanation

Reply:
The 'wayback.py' file was missing a line to load the 'decouple' module, so I added it.